openmc/data/neutron.py

```diff
@@ -34,7 +34,7 @@
 _RESONANCE_ENERGY_GRID = np.logspace(-3, 3, 61)
 
 
-def _get_metadata(zaid, metastable_scheme='nndc'):
+def get_metadata(zaid, metastable_scheme='nndc'):
```

Comment:
No need to make this public. From the update inputs script, you can still access it as `opemc.data.neutron._get_metadata`.